source/extensions/filters/network/http_connection_manager/config.h

```diff
@@ -99,12 +100,15 @@ class InternalAddressConfig : public Http::InternalAddressConfig {
       return unix_sockets_;
     }
 
-    // TODO(snowp): Make internal subnets configurable.
+    if (!cidr_ranges_.isEmpty()) {
```

Comment:
We can have a fast return for the address type isn't IP. Since we needn't search the IP list for the non-IP address, which is wasteful.